libs/model/src/chain-events/ChainEventCreated.command.ts

```diff
@@ -1,17 +1,10 @@
+import { type Command } from '@hicommonwealth/core';
```

Comment:
Let's work on proper naming in the near future. This name in past tense should represent an event.... commands should use imperative names representing potential actions, in this case should be CreateChainEvent... but if the event is already created and we are just reacting to it, this should be a policy or projection